demo/app/ios/Runner/OpenID4CI.swift

```diff
@@ -95,4 +95,24 @@ public class OpenID4CI {
        let resolvedDisplayData = DisplayResolve(vcCredentials, issuerURI, nil, nil)
         return resolvedDisplayData!.serialize(nil)
     }
+    
+    func dynamicRegistrationSupported() throws -> ObjCBool {
+        var dynamicRegistrationSupported: ObjCBool = false
+        try initiatedInteraction.dynamicClientRegistrationSupported(&dynamicRegistrationSupported)
+        return dynamicRegistrationSupported
+    }
+    
+    func dynamicRegistrationEndpoint() throws -> String {
+        var error: NSError?
+        let endpoint = initiatedInteraction.dynamicClientRegistrationEndpoint(&error)
+        if let actualError = error {
+            print("error in dynamic registration endpoint", error!.localizedDescription)
```

Comment:
You should do `actualError.localizedDescription` instead. The `if let` ensures that actualError is not optional.